sdk/servicebus/perf-tests/service-bus/package.json

```diff
@@ -8,7 +8,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/service-bus": "^7.0.0",
+    "@azure/service-bus": "7.8.0",
```

Comment:
I think this is the result of running perf tests on older packages, and should be reverted.